docs/USAGE.md

```diff
@@ -350,7 +349,7 @@ jsf.option('alwaysFakeOptionals', true);
 
 ## Extending dependencies
 
-You may extend [Faker.js](http://marak.com/faker.js/):
+You may extend [Faker.js](https://github.com/Marak/faker.js):
```

Comment:
I couldn't get to `http://marak.com/faker.js/`

Reply:
Thank you!